install/020-MqttBridgeConfigMap.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mqtt-bridge-config
+  labels:
+        app: mqtt-bridge
+data:
+  application.properties: |
+    #Bridge related settings
+    bridge.id=my-bridge
+
+    #MQTT server common
+    mqtt.server.host=localhost
```

```suggestion
    mqtt.server.host=0.0.0.0
```

Using 0.0.0.0 (all interfaces) by default allows the bridge to listen on all interfaces on the pod so accepting traffic from outside. With localhost it's not getting traffic unless the MQTT client is running on the same pod (which is usually not the case :-)).

---

gotcha!